cob_object_detection_msgs/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ add_service_files(
     AcquireObjectImage.srv
     BagTrainObject.srv
     BaTestEnvironment.srv
+    ComputeGraspsVacuumGripper.srv
```

Comment:
Hm, we will need the simple grasp computation tool that we created in order picking in another project, soon, and did not want to copy the message. But it can also be put somewhere else if you have a good suggestion or I just copy the message to the other repository in the worst case.